.changelog/32751.txt

```diff
@@ -0,0 +1,11 @@
+```release-note:enhancement
```

Comment:
Perhaps we can merge these release notes, lets see what @ewbankkit thinks.

Reply:
I have atleast renamed the resource type as I for some reason marked it as a `data-source` while it's a `resource`